### PR TITLE
more rosetta examples

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: docs/source/rosetta.rst [hunk 1]
```diff
@@ -15,39 +15,39 @@ Imports and Construction
 
    # construct a DataFrame
    df = pd.DataFrame({
-      'name': 'Alice Bob Joe Bob'.split(),
-      'amount': np.random.rand(4) * 1000,
-      'id': np.arange(4)
-   })
-
-   df2 = pd.DataFrame({
```
Comment (cpcloud) [inline docs/source/rosetta.rst]: the reason i had `df2` there was to allow people to be able to run the `join` example

Comment (mrocklin) [inline docs/source/rosetta.rst]: The reason I removed it is that it takes up a significant amount of mental space at the beginning of the page for only a single use case.  Seems high-cost per benefit.

A teaching undergrads has made me pretty cynical about peoples' attention spans (perhaps overly-so).

I'll try to find a solution.

Comment (cpcloud) [inline docs/source/rosetta.rst]: okay, i don't have a big problem with it, i just wanted the code examples to be self-contained.

File: docs/source/rosetta.rst [hunk 2]
```diff
@@ -81,16 +77,21 @@ Blaze can simplify and make more readable some common IO tasks that one would wa
...
 +-----------------+-----------------------------------------------------------------+---------------------------------------------------+
-| Reading in a    | .. code-block:: python                                          | .. code-block:: python                            |
-| table from      |                                                                 |                                                   |
-| a SQLite        |    import sqlalchemy as sa                                      |    df = into(pd.DataFrame,                        |
-| database        |    engine = sa.create_engine('sqlite://db.db')                  |              'sqlite://db.db::t')                 |
+| Save result     | .. code-block:: python                                          | .. code-block:: python                            |
+| to CSV file     |                                                                 |                                                   |
+|                 |    df[df.amount < 0].to_csv('output.csv')                       |    into('output.csv', df[df.amount < 0]           |
```
Comment (cpcloud) [inline docs/source/rosetta.rst]: Looks like missing a closing parenthesis here

**Comment by cpcloud** [comment]
going to merge this, fixup the paren and push to the blaze pr. feel free to add more commits in a new pr

**Comment by cpcloud** [comment]
thanks!

**Pull request merged**